packages/record-data/package.json

```diff
@@ -40,6 +41,7 @@
     "ember-cli-terser": "~4.0.1",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.1",
+    "ember-inflector": "^4.0.1",
```

Comment:
Were we needing but missing the inflector?

Reply:
only for testing when we don't bring everything